File: NuGenSurfacePlotter/NuGenSurfacePlotterTestApp/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace NuGenSurfacePlotterTestApp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            nuGenSurfacePlotter1.LoadModel("k:/rshape.3ds");
        }

        private void button2_Click(object sender, EventArgs e)
        {
            nuGenSurfacePlotter1.LoadSurface();
        }
    }
}

File: NuGenIExport/Trial/SampleVB/Genetibase.ApplicationBlocks.DemoVB/Form1.vb
Imports Genetibase.ApplicationBlocks
Imports System
Imports System.Collections.Generic
Imports System.Data
Imports System.Drawing
Imports System.IO
Imports System.Windows.Forms


Public Class Form1


    Private Sub goButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles goButton.Click


        Dim exportForm As NuGenImageExportForm = New NuGenImageExportForm

        Try
            Dim image As Image = image.FromFile(Me.openFileSelector.GetPath)

            exportForm.ShowDialog(image)

        Catch oE As ArgumentException

            MessageBox.Show("Specify the path.", "Alert", MessageBoxButtons.OK, MessageBoxIcon.Exclamation)

        Catch oE As FileNotFoundException

            MessageBox.Show("Specified file could not be found. Make sure the file exists.", "Alert", MessageBoxButtons.OK, MessageBoxIcon.Exclamation)

        End Try

    End Sub
End Class
